storage/EtcCrypttab.cc

```diff
@@ -157,8 +157,10 @@ namespace storage
     }
 
 
-    EtcCrypttab::EtcCrypttab(const string& filename) :
-	ColumnConfigFile()
+    const int EtcCrypttab::DEFAULT_PERMISSIONS = S_IRUSR | S_IWUSR | S_IRGRP; // 0640
```

Comment:
Again, since the comment is the one thing that makes this readable for developers, why not simply use octal 0640 here? That would be much better readable and maintainable.